content/docs/validators/register-vali/_index.md

```diff
@@ -28,6 +28,12 @@ This must be performed on the host machine running the Autonity Go Client, using
 autonity genEnodeProof --nodekey <NODE_KEY_PATH> <TREASURY_ACCOUNT_ADDRESS>
 ```
 
+... or, if you are running the Autonity Go Client in a docker container with the `autonity-chaindata` directory mapped to a direcory of the same name outside the container, you can temporarily create a separate container mapping the same directory for proof generation:
```

Comment:
I do like your suggestion.

As for your reservations: I think the burden is minimal; and since we're describing how to run it in a docker container, then I think it's expected and natural to also describe this step in a docker context. Maybe we can assume that people can figure it out, but it still looks like an omission not to mention it.

As for the easier approach: I thought we are using `--rm` and the folder mapping with Docker precisely so as to keep our examples simple and straightforward, without running into complaints that something doesn't work just because someone forgot to remove an old container; and in order to make it easily possible to access chaindata and config without needing to log into the running container. Personally, I think this is a _good_ solution, and sufficiently easy and straightforward.

Reply:
Point taken about the docker containers. That's not really my field so I have to defer on that one. But I am still uncomfortable copy-pasting big command lines between pages, so I wonder if we can amend the "run autonity" section to add a simple shell script that wraps the long docker command line. This section then just has to suggest the user calls a `. /run-autonity` script with the same params as they'd pass to `autonity`. (With a caveat similar to the one you've given - namely you must have set things up according to the "run autonity" section).